### PR TITLE
Mr 62 horizontal product card doesn´t fit criteria

### PR DESCRIPTION
**Pull request by JaqKent**
fixes in the horizontal card, font size, weight, and background.

### CODE REVIEW

File: src/components/CustomCard/styles.module.scss [hunk 1]
```diff
@@ -28,6 +29,8 @@
 }
 
 .containerSmall {
+    @include flex(flex-start, center, column);
+    padding-top: 1rem;
     @include flex(flex-start, center, column);
     padding-top: 1rem;
```
Comment (nathan-hyan) [inline src/components/CustomCard/styles.module.scss:35]: Why is it doubled?

File: src/components/CustomCard/styles.module.scss [hunk 2]
```diff
@@ -55,6 +58,7 @@
 }
 
 .imgSmall {
+    @include flex(space-between, flex-start, column);
     @include flex(space-between, flex-start, column);
```
Comment (nathan-hyan) [inline src/components/CustomCard/styles.module.scss:62]: Same here and line 70, watch out for these mistakes!

File: src/components/CustomCard/styles.module.scss [hunk 3]
```diff
@@ -94,6 +96,7 @@
 
     width: 15rem;
     height: 4rem;
+    height: 4rem;
```
Comment (nathan-hyan) [inline src/components/CustomCard/styles.module.scss:99]: Same here!

File: src/components/CustomCard/styles.module.scss [hunk 4]
```diff
@@ -136,24 +142,39 @@
     font-weight: 800;
 }
 
+.largeRating {
+    font-size: 1.2rem;
+    font-weight: 500;
+}
+
+
```
Comment (nathan-hyan) [inline src/components/CustomCard/styles.module.scss:150]: Remove extra space

File: src/components/CustomCard/styles.module.scss [hunk 5]
```diff
@@ -136,24 +142,39 @@
...
 .block {
     display: none;
 }
 
 .info {
     @include flex(space-between, flex-start, column);
+
```
Comment (nathan-hyan) [inline src/components/CustomCard/styles.module.scss:157]: Remove extra space

**Review by nathan-hyan — CHANGES_REQUESTED** [review]
Comments!

File: src/components/CustomCard/styles.module.scss (hunk 1 above)
Comment (JaqKent) [reply src/components/CustomCard/styles.module.scss:35]: done

File: src/components/CustomCard/styles.module.scss (hunk 2 above)
Comment (JaqKent) [reply src/components/CustomCard/styles.module.scss:62]: done

File: src/components/CustomCard/styles.module.scss (hunk 3 above)
Comment (JaqKent) [reply src/components/CustomCard/styles.module.scss:99]: done

File: src/components/CustomCard/styles.module.scss (hunk 4 above)
Comment (JaqKent) [reply src/components/CustomCard/styles.module.scss:150]: done

File: src/components/CustomCard/styles.module.scss (hunk 5 above)
Comment (JaqKent) [reply src/components/CustomCard/styles.module.scss:157]: done

**Pull request merged**